src/app/pages/HomePage/index.tsx

```diff
@@ -105,7 +106,7 @@ export const HomePage: FC = () => {
   const { t } = useTranslation()
   const infoAriaLabel = t('home.helpScreen.infoIconAria')
   const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
-  const apiStatusQuery = useGetStatus()
+  const apiStatusQuery = useGetStatus(Network.mainnet) // TODO: what is testnet is off-line? We need a combined status
```

Comment:
typo in the comment

my answer: I'd just check mainnet on homepage. We can cover testnet when user navigates into it https://github.com/oasisprotocol/explorer/issues/356

Reply:
Now that we have proper network selection on this screen, I made it so that we are checking the status of the selected network.